app/code/Magento/InventoryIndexer/Test/Integration/Indexer/SourceItemIndexerTest.php

```diff
@@ -178,4 +199,19 @@ public function reindexAllDataProvider(): array
             ['SKU-3', 30, [GetStockItemDataInterface::QUANTITY => 0, GetStockItemDataInterface::IS_SALABLE => 0]],
         ];
     }
+
+    /**
+     * @param $sku
+     * @param $sourceCode
+     * @return SourceItemInterface
+     */
+    private function getSourceItem($sku, $sourceCode): SourceItemInterface
```

Comment:
Add type hint for params (check all places)

Reply:
@naydav done.